back/infolica/scripts/ldap_query.py

```diff
@@ -11,24 +11,25 @@ class LDAPQuery(object):
 
     @classmethod
     def do_login(cls, request, login, password):
-
+        settings = request.registry.settings
         # Check if user exists in LDAP
         try:
-            connector = get_ldap_connector(request)
+            server = Server(settings['ldap_url'])
+            with Connection(server=server, user="ACN\\"+login, password=password, auto_bind=True, authentication=NTLM) as conn:
```

Comment:
Oh nice... so you connect to LDAP with the current user and you make him find himself ;-)